src/hardware/ipxserver.cpp

```diff
@@ -98,22 +94,19 @@ static void sendIPXPacket(Bit8u *buffer, Bit16s bufSize) {
 		}
 	} else {
 		// Specific address
-		for(i=0;i<SOCKETTABLESIZE;i++) {
+		for (uint16_t i = 0; i < SOCKETTABLESIZE; i++) {
 			if((connBuffer[i].connected) && (ipconn[i].host == desthost) && (ipconn[i].port == destport)) {
 				outPacket.address = ipconn[i];
-				result = SDLNet_UDP_Send(ipxServerSocket,-1,&outPacket);
-				if(result == 0) {
+				const int result = SDLNet_UDP_Send(ipxServerSocket,
+				                                   -1, &outPacket);
```

Comment:
This `-1` is a bit of a magic number that's used in the client and server IPX code.
from the SDL docs, this argument is the channel described as:
`If channel is not -1 then the packet is sent to all the socket channels bound addresses. `

Suggest we define this in a header as `UDP_UNICAST = -1` or as a `constexpr`.

Reply:
Hmm, documentation on my machine says something different:
```
Send a single packet to the specified channel.
If the channel specified in the packet is -1, the packet will be sent to
the address in the 'src' member of the packet.
```

Reply:
Yup :-) `-1` means unicast.
 - the documentation I mentioned say _" != -1 then send to **all** socket channels"_
 - yours says "== -1 send to the src of the packet" (which is unicast)

Reply:
Done :)